internal/tui/components/chat/editor/editor.go

```diff
@@ -560,3 +567,44 @@ func New(app *app.App) Editor {
 
 	return e
 }
+
+func (m *editorCmp) getUserMessagesAsText(ctx context.Context) ([]string, error) {
+	allMessages, err := m.app.Messages.List(ctx, m.session.ID)
```

Comment:
we need to create a new command here to list all the messages regardless of the session and maybe filter them by type and order by create date, this way you can get messages from previous sessions too